<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B4E308O" failures="2" tests="8" name="RetryLogicSeconWay" time="0.078" errors="0" timestamp="2021-09-06T21:34:53 IST">
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestOne" time="0.008">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestOne -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestOne" time="0.001">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestOne -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestOne" time="0.002">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestOne -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestOne" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at RetryFailedTestCase.RetryLogicSecondWay.retrySecondLogicTestOne(RetryLogicSecondWay.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- retrySecondLogicTestOne -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestTwo" time="0.001">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestTwo -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestTwo" time="0.001">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestTwo -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestTwo" time="0.001">
    <skipped/>
  </testcase> <!-- retrySecondLogicTestTwo -->
  <testcase classname="RetryFailedTestCase.RetryLogicSecondWay" name="retrySecondLogicTestTwo" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at RetryFailedTestCase.RetryLogicSecondWay.retrySecondLogicTestTwo(RetryLogicSecondWay.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- retrySecondLogicTestTwo -->
</testsuite> <!-- RetryLogicSeconWay -->
